.github/workflows/ci.yml

```diff
@@ -686,14 +686,19 @@ jobs:
         include:
           # Litentry
           - test_name: lit-di-substrate-identity-test
+            overridden_test_name: '[substrate] tee-single-worker-test(lit-di-identity)'
```

Comment:
what do you think if we will call it **label** instead? Otherwise we will get a question, why it's overridden

Reply:
what's the benefit of "overridden" ?

Reply:
it helps to see which tests failed. Otherwise it's hidden or difficult to read
<img width="950" alt="Screenshot 2024-04-12 at 17 40 19" src="https://github.com/litentry/litentry-parachain/assets/2113554/e1c87d7d-6903-49df-96cd-ae3823dd16dc">

Reply:
per Igor's suggest, I renamed overridden to label now, used in the above graph to show important text first without being hidden